app/core/ingestion/family.py

```diff
@@ -183,6 +186,9 @@ def none_if_empty(data: str) -> Optional[str]:
                 params.document_title,
                 family_document.physical_document,
             )
+            update_physical_document_languages(
+                db, params.language, result, family_document.physical_document
+            )
```

Comment:
Is deleting all existing languages on every ingest not a little heavy handed? I had wondered if we could label all existing languages as not visible with a one-off script, then this script method would only need to add a missing language or mark an existing one as "user + visible" if necessary... 

Also, this makes it impossible for the pipeline to add document languages unless we run a complete update each time (currently we only send updates).

Reply:
I don't believe the code does delete all existing languages on every ingest - perhaps you can point out where you think this is occurring?